osquery/events/linux/auditdnetlink.cpp

```diff
@@ -60,16 +61,30 @@ DECLARE_bool(audit_allow_process_events);
 DECLARE_bool(audit_allow_sockets);
 DECLARE_bool(audit_allow_user_events);
 DECLARE_bool(audit_allow_selinux_events);
+DECLARE_bool(audit_allow_apparmor_events);
 
 // user messages should be filtered
 // also, we should handle the 2nd user message type
 namespace {
 bool IsSELinuxRecord(const audit_reply& reply) noexcept {
```

Comment:
This isn't cased correctly, do you think you could fix this to follow the standard osquery convention of camelCaseFunctionNames?

Reply:
@alessandrogario

Reply:
Hello @obelisk!

The old audit framework (both the publishers and the subscribers) already used different conventions like `function_name()`, `functionName()` and `FunctionName()`. I ended up converting them all to the third form, but now I can see the the first one was the right call! We can open an issue to correct this.

Reply:
Alright, does it mean I can leave it as is because it will be corrected in a seperate issue?